src/widgets/simulator.rs

```diff
@@ -174,3 +171,19 @@ impl<'a> Widget for Simulator<'a> {
         .response
     }
 }
+
+fn get_image_path(selected_job: u8, action: Action) -> String {
+    let base_url = if cfg!(target_arch = "wasm32") {
+        env!("BASE_URL")
```

Comment:
Use 2 separate `let base_url = ...` with `#[cfg(...)]` so that the native target can build without the `BASE_URL` env variable.

Reply:
Good catch. I found that probably makes more sense to turn these into constants rather than using a function https://github.com/Human-Hamster/raphael-rs/commit/6147511bd4377d448f512b28cd27bf554d8c6459

(Ignore the white-space changes, I undid those in the latest commit).